### ISSUE TITLE
[DEVICE REQUEST] SilverCrest doorbell

### ISSUE THREAD

**Issue by Fredo70** [dresden-elektronik/deconz-rest-plugin#4260]
## Device

- Product name: Smart Home SilverCrest Kabellose Türklingel (model N° on the back of the device HG06668)
- Manufacturer: _TZ1800_ladpngdx
- Model identifier: TS0211
- Device type : Please remove all unrelated device types. 
  - Other: doorbell

## Screenshots

<img width="882" alt="clusterinfo" src="https://user-images.githubusercontent.com/6943043/105608007-adaf5980-5da1-11eb-94f8-99d02f0b8e6a.png">
<img width="266" alt="nodeinfo" src="https://user-images.githubusercontent.com/6943043/105608009-b0aa4a00-5da1-11eb-9cdc-1edbab7527c6.png">

**Comment by ebaauw** [comment]
Please list the _IAS Zone_ cluster.  Make sure to read the attributes, pressing the device (or possibly its reset button) to wake it.

**Comment by Fredo70** [comment]
<img width="882" alt="clusterinfo" src="https://user-images.githubusercontent.com/6943043/105608357-d3d5f900-5da3-11eb-9ff0-a25be8512211.png">

**Comment by Fredo70** [comment]
<img width="356" alt="IAS_Zone" src="https://user-images.githubusercontent.com/6943043/105608565-e7815f80-5da3-11eb-8ce3-7453c29783ef.png">

**Comment by Mario-75** [comment]
I have the same doorbell but it does not show up at all on my ConBee 2.
I have Rpi4 with Openhabian ("Raspbian GNU/Linux 10 (buster)"). I installed the latest firmware (26660700) and used beta driver (2.09.01). The gateway seems work since it finds my new switches (SilverCrest Zwischenstecker) and I can control them with openHAB.
I did another installation on my Win10 computer with GUI, moved the ConBee2 stick to the machine, but same situation. Doorbell does not show up when pressing the connect button (3x within 2s).
I tried the loging (/usr/bin/deCONZ --dbg-info=2 > dbg-info2.txt) but I don’t really know what to look for.
